src/Explorer/Controls/CommandButton/CommandButtonComponent.tsx

```diff
@@ -259,7 +260,18 @@ export class CommandButtonComponent extends React.Component<CommandButtonCompone
           onClick={(e: React.MouseEvent<HTMLSpanElement>) => this.commandClickCallback(e)}
         >
           <div className={contentClassName}>
-            <img className="commandIcon" src={this.props.iconSrc} alt={this.props.iconAlt} />
+            if (this.props.iconName){" "}
+            {
+              <div>
+                <Icon
+                  styles={{ root: { marginBottom: -3 } }}
+                  className="panelInfoIcon"
+                  iconName={this.props.iconName}
+                  ariaLabel="ChatBot"
```

Comment:
`CommandButtonComponent` is a generic button class used for all kinds of buttons so having special handling for chatbot button seems odd. Can we generalize this instead?